src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -292,10 +293,20 @@ public IActionResult GetAdminToken()
         [HttpGet]
         [HttpPost]
         [Route("admin/warmup")]
-        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
         [RequiresRunningHost]
         public async Task<IActionResult> Warmup([FromServices] WebJobsScriptHostService hostService)
         {
+            // Endpoint only for Windows Elastic Premium or Linux App Service plans
+            if (!(_environment.IsLinuxAppService() || _environment.IsWindowsElasticPremium()))
```

Comment:
Wondering why you have the IsLinuxAppService() check. Both my Windows and Linux Elastic Premium apps have sku "ElasticPremium". Why isn't checking for just that sku isn't sufficient?

Reply:
Does Linux Elastic Premium have WEBSITE_SKU environment variable set to ElasticPremium? I did not find that information in my sample app. Perhaps, I looked it incorrectly?

Reply:
Look at my apps functiondev-mathewc-eptest and functiondev-mathewc-eptest-linux in Observer

Reply:
The site config will have the sku. But that information isn't available to the Host. Unless there's already a way to do that?
I thought Host relies on the environment variables.

Reply:
Rigth, and I assumed in Linux the same WEBSITE_SKU would have the ElasticPremium value like windows does. However, I just checked and on linux it the environment variable doesn't even exist - which surprised me.